extensions/typescript-language-features/src/languageFeatures/organizeImports.ts

```diff
@@ -144,7 +148,18 @@ export function register(
 			API.v430,
 			vscode.CodeActionKind.Source.append('sortImports'),
 			localize('sortImportsAction.title', "Sort Imports"),
-			true,
+			OrganizeImportsMode.SortAndCombine,
+			commandManager,
+			fileConfigurationManager,
+			telemetryReporter,
+			selector
+		),
+		ImportsCodeActionProvider.register(
+			client,
+			API.v490,
+			vscode.CodeActionKind.Source.append('removeUnusedImports'),
+			localize('removeUnusedImportsAction.title', "Remove Unused Imports"),
```

Comment:
@mjbvz if I wanted to expose these code actions as commands in the command palette, what’s the best way to wire that up?

Reply:
@andrewbranch You'll need to register a new command for this. You can look at the `typescript.goToSourceDefinition` for an example of this or I can help our with this part of the change